Mobile Buy SDK/Mobile Buy SDK/Buy.h

```diff
@@ -65,6 +65,7 @@ FOUNDATION_EXPORT const unsigned char BuyVersionString[];
 #import <Buy/BUYClient.h>
 #import <Buy/BUYClient+Customers.h>
 #import <Buy/BUYClient+Checkout.h>
+#import <Buy/BUYClient+CheckoutHelpers.h>
```

Comment:
Used to only exist in Merchant App, so yeah, maybe we should do something nicer. If we are going to improve it, then `-handleCheckout:` is also crappy name which should instead be `-updateOrCreateCheckout:completion:`.